File: CS/MvcDashboard_ServerSideApi/Controllers/DefaultDashboardController.cs
using DevExpress.DashboardWeb;
using DevExpress.DashboardWeb.Mvc;

namespace MvcDashboard_ServerSideApi.Controllers {
    public class DefaultDashboardController : DashboardController {
        public DefaultDashboardController(DashboardConfigurator dashboardConfigurator) 
            : base(dashboardConfigurator) {
            
        }
    }
}

File: VB/MvcDashboard_ServerSideApi/Controllers/DefaultDashboardController.vb
Imports DevExpress.DashboardWeb
Imports DevExpress.DashboardWeb.Mvc

Namespace MvcDashboard_ServerSideApi.Controllers
	Public Class DefaultDashboardController
		Inherits DashboardController

		Public Sub New(ByVal dashboardConfigurator As DashboardConfigurator)
			MyBase.New(dashboardConfigurator)

		End Sub
	End Class
End Namespace